docs/blogs/EnhancedDevPreviewGatewayAPI/GettingCreative.md

```diff
@@ -0,0 +1,263 @@
+# Getting Creative
+
+In the previous section, we showed how to create a simple HTTPRoute. Let’s expand upon this example and include more
+advanced usages of Gateway API.
+
+## Route Matching
+Gateway API is a powerful API for directing traffic based on request characteristics such as path, headers, query
+parameters, and HTTP methods. Let’s explore how we can use these functions to direct traffic to different backends.
+
+#### Path Based Routing
+In this first example, we will match traffic based on the paths `/foo` and `/bar`. Path-based routing is useful for
+segmenting a website into various backend services. In the current OpenShift Route API, we must make a Route per path:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http-foo
+  namespace: demo-app
+spec:
+  host: http.gwapi.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+  path: /foo
+---
+apiVersion: v1
+kind: Route
+metadata:
+  name: http-bar
+  namespace: demo-app
+spec:
+  host: http.${DOMAIN}
+  to:
+    kind: Service
+    name: bar-app
+  port:
+    targetPort: 8080
+  path: /bar
+```
+
+However, Gateway API enables the path-based matching to be declared in a single HTTPRoute object as we will demonstrate
+in this example.
+
+First, let’s create another demo application deployment and service using `oc new-app`, our `cakephp-ingress-demo`
+project, and the `bar` git branch. This will be used as an alternate server backend, `bar-app`, to direct traffic to:
+```shell
+$ oc new-app -n demo-app --name bar-app https://github.com/openshiftdemos/cakephp-ingress-demo#bar
+$ oc rollout status deployment -w -n demo-app bar-app
+```
```

Comment:
````suggestion
First, let’s create another demo application deployment and service using the `oc new-app` command and the `bar` branch of our `cakephp-ingress-demo` GitHub
project. This will be used as an alternate server backend, `bar-app`, to direct traffic to:
```console
$ oc new-app -n demo-app --name bar-app https://github.com/openshiftdemos/cakephp-ingress-demo#bar
$ oc rollout status deployment -w -n demo-app bar-app
[expected output]
```
````
What is the expected output?

Reply:
Added expected output (to both, just to be consistent).